### PR TITLE
DFR-3086  CS: Send Order: Unapproved order not sent to unrepresented respondent

### PR DESCRIPTION
**Pull request by hamzahtahirhmcts**
### Jira link (if applicable)
(https://tools.hmcts.net/jira/browse/DFR)[DFR-3086](https://tools.hmcts.net/jira/browse/DFR-3086)

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Comment by hamzahtahirhmcts** [comment]
I'm getting 4 failed tests that are returning null pointer exceptions. Can't quite figure out the cause for this. Any ideas?

**Comment by hamzahtahirhmcts** [comment]
Respondent docs on preview env:

Screenshot 2024-06-13 at 15 04 34

Applicant docs on preview env:
Screenshot 2024-06-13 at 15 06 02

**Review by ptrelease — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentOrderNotApprovedDocumentService.java [hunk 1]
```diff
@@ -48,11 +48,11 @@ public List<BulkPrintDocument> prepareApplicantLetterPack(FinremCaseDetails case
 
         List<BulkPrintDocument> documents = new ArrayList<>();
 
-        documents.add(coverLetter(caseDetails, authorisationToken));
+        documents.add(notApprovedCoverLetter(caseDetails, authorisationToken, APPLICANT));
         addEitherNotApprovedOrderOrGeneralOrderIfApplicable(caseDetails, documents, authorisationToken);
 
         return documents.size() == 1
-            ? emptyList()  // if only cover letter then print nothing
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentOrderNotApprovedDocumentService.java:55]: That did seem like a useful comment to me.

Comment (hamzahtahirhmcts) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentOrderNotApprovedDocumentService.java:55]: Added back [e180735](https://github.com/hmcts/finrem-case-orchestration-service/pull/1732/commits/e1807355486a1c68b9f813c423ea8033bb9d520d)

**Review by jthmcts — DISMISSED** [review]

**Comment by hamzahtahirhmcts** [comment]
`FinremCaseDataTest.testContestedConfigFinRemCaseData failed`

**Review by al-hmcts — APPROVED** [review]

**Review by al-hmcts — APPROVED** [review]

**Review by so99y — APPROVED** [review]

**Pull request merged**